pkg/vm/compiler/compiler.go

```diff
@@ -82,17 +81,15 @@ func CompileAndSave(src string, o *Options) error {
 	}
 	b, err := ioutil.ReadFile(src)
 	if err != nil {
-		return err
+		return "", err
 	}
 	b, err = Compile(bytes.NewReader(b))
 	if err != nil {
-		return fmt.Errorf("error while trying to compile smart contract file: %v", err)
+		return "", fmt.Errorf("error while trying to compile smart contract file: %v", err)
 	}
 
-	log.Println(hex.EncodeToString(b))
-
 	out := fmt.Sprintf("%s.%s", o.Outfile, o.Ext)
-	return ioutil.WriteFile(out, b, os.ModePerm)
+	return hex.EncodeToString(b), ioutil.WriteFile(out, b, os.ModePerm)
```

Comment:
Maybe it'd be better to hexify it in the caller and return plain bytes here.

Reply:
done